website/docs/reference/resource-configs/snowflake-configs.md

```diff
@@ -9,6 +9,373 @@ To-do:
 - use the reference doc structure for this article / split into separate articles
 --->
 
+## Iceberg table format <Lifecycle status="beta"/>
+
+Snowflake Iceberg tables support for the dbt-snowflake adapter is available for three out-of-the-box materializations:  
```

Comment:
maybe a small little intro of iceberg would be beneficial for users? e..g:

Snowflake Iceberg helps manage large datasets in Snowflake efficiently. Snowflake Iceberg tables support the dbt-snowflake adapter, which you can use for three out-of-the-box materializations: